server/endpoint/endpoint.go

```diff
@@ -0,0 +1,62 @@
+package endpoint
+
+import (
+	"github.com/giantswarm/microendpoint/endpoint/healthz"
+	"github.com/giantswarm/microendpoint/endpoint/version"
+	healthzservice "github.com/giantswarm/microendpoint/service/healthz"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+
+	"github.com/giantswarm/node-operator/server/middleware"
+	"github.com/giantswarm/node-operator/service"
+)
+
+// Config represents the configuration used to create a endpoint.
+type Config struct {
+	// Dependencies.
+	Logger     micrologger.Logger
+	Middleware *middleware.Middleware
+	Service    *service.Service
+}
+
+// Endpoint is the endpoint collection.
+type Endpoint struct {
+	Healthz *healthz.Endpoint
+	Version *version.Endpoint
+}
+
+// New creates a new configured endpoint.
+func New(config Config) (*Endpoint, error) {
+	var err error
+
+	var healthzEndpoint *healthz.Endpoint
+	{
+		healthzConfig := healthz.DefaultConfig()
```

Comment:
Maybe shorten `healthzConfig` to `c`? There's also another one below.

Reply:
Done.